<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cddoimk_form</class>
 <widget class="QMainWindow" name="cddoimk_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Đổi mật khẩu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(224, 243, 253);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(154, 205, 244);
font: 16pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Tài khoản và bảo mật&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_return">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(154, 205, 244);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/ic_return.png</normaloff>images/ic_return.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_doimk">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Cambria&quot;;
background-color: rgb(146, 214, 255);</string>
    </property>
    <property name="text">
     <string>Đổi mật khẩu</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>99</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nhập mật khẩu mới</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txt_mkmoi">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Mật khẩu mới</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nhập mật khẩu hiện tại</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nhập lại mật khẩu mới</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txt_mkcu">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Mật khẩu cũ</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txt_nhaplaimk">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Nhập lại mật khẩu mới</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_huybo">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 194, 209);</string>
       </property>
       <property name="text">
        <string>Hủy bỏ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_luu">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 194, 209);</string>
       </property>
       <property name="text">
        <string>Lưu thay đổi</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_hienmk">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>113</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(245, 40, 145, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/ic_visible.png</normaloff>images/ic_visible.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_anmk">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>113</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(245, 40, 145, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/ic_hide.png</normaloff>images/ic_hide.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_anmk_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(245, 40, 145, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/ic_hide.png</normaloff>images/ic_hide.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_anmk_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>185</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(245, 40, 145, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/ic_hide.png</normaloff>images/ic_hide.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_hienmk_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(245, 40, 145, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/ic_visible.png</normaloff>images/ic_visible.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_hienmk_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>185</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(245, 40, 145, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/ic_visible.png</normaloff>images/ic_visible.png</iconset>
    </property>
   </widget>
   <zorder>btn_anmk</zorder>
   <zorder>label</zorder>
   <zorder>btn_return</zorder>
   <zorder>lbl_doimk</zorder>
   <zorder>formLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>btn_hienmk</zorder>
   <zorder>btn_anmk_2</zorder>
   <zorder>btn_anmk_3</zorder>
   <zorder>btn_hienmk_2</zorder>
   <zorder>btn_hienmk_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
